packages/now-ruby/test/fixtures/05-sinatra/api/index.ru

```diff
@@ -1,3 +1,3 @@
-require './app'
+require_relative 'app'
```

Comment:
I think the `app.rb` file should not live in `/api` or else it will be turned into a serverless function.

Perhaps it should be `/src/app.rb` or maybe `/api/_lib/app.rb`?